internal/locale/locales/en-us.yaml

```
@@ -118,7 +118,7 @@ flag_state_auth_username_description:
 flag_state_auth_password_description:
   other: The password to authenticate with
 flag_state_auth_totp_description:
-  other: The TOTP code generated via two-factor authentication
+  other: The TOTP code generated via two-factor authentication (used in conjunction with username and password flags)
```

Maybe we should add something about how it _has_ to be used with the other flags. Below is just a suggestion.
```suggestion
  other: The TOTP code generated via two-factor authentication (must be used in conjunction with username and password flags)
```